File: startingPoint/02-EstruturaSequencial/Exercicios_Propostos/07.py
#!/usr/bin/python
# coding: latin-1

# 07 - Faça um programa que receba o valor de um depósito e o valor da taxa de juros, calcule
#e mostre o valor do rendimento e o valor total depois do rendimento

dep = float(input("Digite o valor do depósito aqui: "))
taxa = float(input("Digite o valor da taxa de juros: "))

rend = dep * taxa / 100

total = (dep + rend)

print("O valor do rendimento será: R$", rend, "O valor total após o rendimento será: R$", total)

File: SRC - PERL/00 - Estruturas Sequenciais/07.pl
#!/usr/bin/env perl
use v5.10;

use strict;
use warnings;

use utf8;
use open qw(:std :utf8);

# Faca um programa que receba o valor de um deposito e o valor da taxa de juros, calcule e mostre o valor do rendimento e o valor total depois do rendimento

print "Digite o valor do despósito > ";
my $desposito = <>;
chomp $desposito;

print "Digite o valor da taxa de juros > ";
my $taxa_juros = <>;
chomp $taxa_juros;

my $rendimento = $desposito * $taxa_juros / 100;
my $valor_total = $desposito + $rendimento;

printf "O valor total é R\$ %.2f ", $valor_total;
say "";
